src/styles/theme.tsx

```diff
@@ -30,6 +30,9 @@ const materialUiOverrides = {
     iconSizeMedium: {
       '& > *:first-child': {
         fontSize: 12
+      },
```

Comment:
Lines 34-36 describe the same selector as lines 31-33. I think I like using `fontSize: 14` better, so lets change this section to:

```js
const materialUiOverrides = {
  MuiButton: {
    root: {
      borderRadius: 0
    },
    label: {
      fontWeight: 600
    },
    iconSizeMedium: {
      '& > *:first-child': {
        fontSize: 14
      }
    },
    iconSizeSmall: {
      '& > *:first-child': {
        fontSize: 14
      }
    }
  },
  ...
}
```

Reply:
Updated.